elixir/mix.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Web.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :web,
+      version: "0.1.0",
+      elixir: "~> 1.15",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {Web.Application, []}
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:plug_cowboy, "> 0.0.0"},
```

Comment:
@everzet Is there a way to install deps from the command line where the current version ends up here automatically? (similar to e.g. npm)